locale/af/common.json

```diff
@@ -22,5 +22,6 @@
   "slack.link.url": "slack.link.url",
   "github.link.text": "github.link.text",
   "github.link.url": "github.link.url",
-  "opens.new.window": "opens.new.window"
+  "opens.new.window": "opens.new.window",
+  "workInProgressWarning": "workInProgressWarning"
```

Comment:
Is there a reason why the ones above use dot syntax and the added one camel case?

Reply:
No. I apparently just didn't pay enough attention to the previous ones when making the new entry :) Will fix it.